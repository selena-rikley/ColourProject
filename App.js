import React from 'react';
import { StyleSheet, View } from 'react-native';
import { createAppContainer, createSwitchNavigator } from 'react-navigation';
import ErrorBoundary from './components/ErrorBoundary';
import HomeScreen from './screens/HomeScreen';

const MainNavigator = createSwitchNavigator({
  Home: HomeScreen
});

const AppContainer = createAppContainer(MainNavigator);

const App = () => (
  <ErrorBoundary>
    <View style={styles.container}>
      <AppContainer />
    </View>
  </ErrorBoundary>
);

export default App;

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center'
  }
});
