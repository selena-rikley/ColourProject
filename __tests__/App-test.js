import React from 'react';
import renderer from 'react-test-renderer';

import App from '../App';

describe('App', () => {
  it('renders the root without loading screen', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
