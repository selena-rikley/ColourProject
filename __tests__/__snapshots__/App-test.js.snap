// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders the root without loading screen 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
      }
    }
  >
    <RCTScrollView
      contentContainerStyle={
        Object {
          "paddingTop": 30,
        }
      }
      style={
        Object {
          "backgroundColor": "#fff",
          "flex": 1,
        }
      }
    >
      <View>
        <View
          style={
            Object {
              "alignItems": "center",
              "marginBottom": 20,
              "marginTop": 10,
            }
          }
        >
          <Image
            source={1}
            style={
              Object {
                "height": 80,
                "marginLeft": -10,
                "marginTop": 3,
                "resizeMode": "contain",
                "width": 100,
              }
            }
          />
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "marginHorizontal": 50,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "rgba(0,0,0,0.4)",
                "fontSize": 14,
                "lineHeight": 19,
                "marginBottom": 20,
                "textAlign": "center",
              }
            }
          >
            Development mode is enabled: your app will be slower but you can use useful development tools.
            <Text
              onPress={[Function]}
              style={
                Object {
                  "color": "#2e78b7",
                  "fontSize": 14,
                }
              }
            >
              Learn more
            </Text>
          </Text>
          <Text
            style={
              Object {
                "color": "rgba(96,100,109, 1)",
                "fontSize": 17,
                "lineHeight": 24,
                "textAlign": "center",
              }
            }
          >
            Get started by opening
          </Text>
          <View
            style={
              Array [
                Object {
                  "backgroundColor": "rgba(0,0,0,0.05)",
                  "borderRadius": 3,
                  "paddingHorizontal": 4,
                },
                Object {
                  "marginVertical": 7,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  undefined,
                  Object {
                    "fontFamily": "space-mono",
                  },
                ]
              }
            />
          </View>
          <Text
            style={
              Object {
                "color": "rgba(96,100,109, 1)",
                "fontSize": 17,
                "lineHeight": 24,
                "textAlign": "center",
              }
            }
          >
            Change this text and your app will automatically reload.
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "marginTop": 15,
            }
          }
        >
          <View
            accessible={true}
            isTVSelectable={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
                "paddingVertical": 15,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#2e78b7",
                  "fontSize": 14,
                }
              }
            >
              Help, it didnâ€™t automatically reload!
            </Text>
          </View>
        </View>
      </View>
    </RCTScrollView>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fbfbfb",
          "bottom": 0,
          "left": 0,
          "paddingVertical": 20,
          "position": "absolute",
          "right": 0,
          "shadowColor": "black",
          "shadowOffset": Object {
            "height": -3,
            "width": 0,
          },
          "shadowOpacity": 0.1,
          "shadowRadius": 3,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "rgba(96,100,109, 1)",
            "fontSize": 17,
            "textAlign": "center",
          }
        }
      >
        This is a tab bar. You can edit it in:
      </Text>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "rgba(0,0,0,0.05)",
              "borderRadius": 3,
              "paddingHorizontal": 4,
            },
            Object {
              "marginTop": 5,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "rgba(96,100,109, 0.8)",
              },
              Object {
                "fontFamily": "space-mono",
              },
            ]
          }
        />
      </View>
    </View>
  </View>
</View>
`;
